SQL/paradise_schema.sql

```diff
@@ -266,6 +266,7 @@ CREATE TABLE `player` (
   `exp` mediumtext,
   `clientfps` smallint(4) DEFAULT '0',
   `atklog` smallint(4) DEFAULT '0',
+  `ambientocclusion` smallint(4) DEFAULT '1',
```

Comment:
If this is just a binary toggle, this doesnt need a full SQL change, just utilise the existing toggles system we have in place with flags

Reply:
We’re out of preference toggles, I’m pretty sure. We already have issues with existing toggles not saving. Refactoring that is out of scope.

Reply:
Quoted from another PR
```
DB Flag analysis. There are 4 ways of storing an integer in a MySQL database, 5 if you count text. The types are as follows:

BIGINT: 9,223,372,036,854,775,807 (Can store 63 flags)
INT: 2,147,483,647 (Can store 31 flags) | This is what we currently use
SMALLINT: 32,767 (Can store 15 flags)
TINYINT: 255 (Can store 7 flags)

In our code we currently use 18 flags, with 13 to spare
```
The reason saving/loading didnt work is because someone forgot to increase the sanitize values when saving and loading, so it tried to load but the value was too big, and wasnt hapyp.

Reply:
Aren’t bitflags in BYOND limited to 16 bits though? Or did that get changed?

Reply:
That got changed probably, since were using 18 flags

Reply:
And we’re sure the last few toggles we added are actually saving now, so it was just an SQL issue? Especially because the fancy NanoUI option was also added as a regular preference instead of a toggle.

Reply:
I am sure they are saving, as other PRs since then have been made with no issue, iirc